Documentation/building/cross-building.md

```diff
@@ -91,4 +91,24 @@ The output is at bin/<BuildOS>.AnyCPU.Debug.
     drwx------  33 lgs lgs  4096  3  6 21:33 Windows_NT.AnyCPU.Debug
     drwx------  33 lgs lgs  4096  3  6 19:36 Windows_NT.AnyCPU.Release
 
+Building corefx for Linux ARM Emulator
+=======================================
 
+It is possible to build corefx binaries (native and managed) for the Linux ARM Emulator (latest version provided here: [#3805](https://github.com/dotnet/coreclr/issues/3805)).
+The `scripts/arm32_ci_script.sh` script does this.
+
+The following instructions assume that:
+* You have setup the extracted emulator at `/opt/linux-arm-emulator` (such that `/opt/linux-arm-emulator/platform/rootfs-t30.ext4` exists)
+* The mount path for the emulator rootfs is `/opt/linux-arm-emulator-root` (change this path if you have a working directory at this path).
```

Comment:
The script has been tested on Ubuntu. But AFAIK the script should run in any distro of Linux.